pkgs/native_toolchain_c/lib/src/native_toolchain/gcc.dart

```diff
@@ -29,6 +29,15 @@ final i686LinuxGnuGccAr = _gnuArchiver('i686-linux-gnu');
 /// [gnuLinker] with [Tool.defaultResolver] for [Architecture.ia32].
 final i686LinuxGnuLd = _gnuLinker('i686-linux-gnu');
 
+/// [gcc] with [Tool.defaultResolver] for [Architecture.x64].
+final x86_64LinuxGnuGcc = _gcc('x86_64-linux-gnu');
```

Comment:
What happens on x64 hosts? Can one install the `x86_64-linux-gnu-gcc` on an x64 host? If so, will it be an alias for `gcc`?

Reply:
On an X64 Debian machine, if I install `g++` I get both `/usr/bin/g++` and `/usr/bin/x86_64-linux-gnu-g++`.